clipper_admin/clipper_admin/kubernetes/prom_configmap.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
```

Comment:
What is the configmap for?

Reply:
configMap stores Prometheus configuration data in Kubernetes. Kubernetes then pass the configuration to the prometheus pod when the pod is spawned.